bin/test.bash

```diff
@@ -23,17 +25,63 @@ test_apps_wc() {
         "${pipeline_path}/test/services/test-wc.sh" "${config[config_file_wc]}"
 }
 
-#
-# ENTRYPOINT
-#
-
-config_load "$1"
-if [[ $1 == "wc" ]]; then
-    test_apps_wc
-elif [[ $1 == "sc" ]]; then
-    test_apps_sc
-else
-    echo "ERROR:  [$1] is an invalid argument:"
-    echo "usage:   ck8s test <wc|sc>. "
-    exit 1
-fi
+function sc_help() {
+    printf "%s\n" "[Usage]: test sc [target] [ARGUMENTS]"
+    printf "%s\n" "List of targets:"
+    printf "\t%-23s %s\n" "opensearch" "Open search checks"
+    printf "\t%-23s %s\n" "fluentd" "Fluentd checks"
+    printf "\t%-23s %s\n" "opensearch-dashboards" "Opensearch dashboards checks"
```

Comment:
Maybe we can delete these since they don't exists

```suggestion
```

Reply:
Sure